### PR TITLE
Fix cbindgen

### PR DESCRIPTION
**Pull request by JJTech0130**
This should hopefully fix it. I'm really not sure what's going on with the documentation_length thing,
but it doesn't seem to be necessary anyway, so I just commented it out.

### CODE REVIEW

File: cbindgen.toml [hunk 1]
```diff
@@ -28,7 +28,7 @@ line_length = 500
 tab_width = 2
 documentation = true
 documentation_style = "auto"
-documentation_length = "full"
+#documentation_length = "full"
```
Comment (jkcoxson) [inline cbindgen.toml:31]: What was this removed for?

**Comment by jkcoxson** [comment]
Does this still read from the configuration toml?

**Comment by JJTech0130** [comment]
yes

**Comment by JJTech0130** [comment]
(you can tell because if you un-comment out that line it breaks)

**Comment by JJTech0130** [comment]
honestly, it seems like a cbindgen bug, but I don't care enough to investigate

**Review by Bratan613 — COMMENTED** [review]
[]()

**Pull request merged**